### ISSUE TITLE
Error in as.Date.numeric(value) : 'origin' must be supplied

### ISSUE THREAD

**Issue by dennet** [hafen/trelliscopejs#42]
First of all, love the package! An awesome tool! Thanks a lot.

After the latest update I'm getting the following error when my x-asis is a Date type variable: 
Error in as.Date.numeric(value) : 'origin' must be supplied

Simple code example: 

```
library(tidyverse)
library(trelliscopejs)

economics_long %>% 
    ggplot(aes(date, value)) +
    geom_line() +
    facet_trelliscope(~variable)
```

**Comment by hafen** [comment]
Thanks for the report. In the latest version there were some updates for allowing you to specify scales for ggplot faceting (and the default is now for all axes to have the same limits as it should be) and there is some computation in the background to set these. Can you provide a reproducible example (provide a "dummy" data set that reproduces the error)?

@schloerke can you look into this?

**Comment by dennet** [comment]
Thank you for looking into it. The snippet I've shared  should be reproducible; economics_long dataset is part of ggplot2.

**Comment by hafen** [comment]
Oops I should have realized that :). Thanks.

**Comment by schloerke** [comment]
Fixed in #43

**Comment by dennet** [comment]
thank you, guys! really appreciate the quick turnaround!

**Comment by asad-taj** [comment]
I am still facing the same issue

**Comment by hafen** [comment]
Can you provide an example? The originally-provided example is now working with the fix in #43.
